src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,139 @@
-            </a>
-          </td>
-          <td>f</td>
-          <td>1835</td>
-          <td>1917</td>
-          <td>Seraphina Spelier</td>
-          <td>Charles Sturm</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/emma-de-milliano-1876"
-            >
-              Emma de Milliano
-            </a>
-          </td>
-          <td>f</td>
-          <td>1876</td>
-          <td>1956</td>
-          <td>Sophia van Damme</td>
-          <td>Petrus de Milliano</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/emile-haverbeke-1877">Emile Haverbeke</a>
-          </td>
-          <td>m</td>
-          <td>1877</td>
-          <td>1968</td>
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carolus-haverbeke-1832">Carolus Haverbeke</a>
-          </td>
-        </tr>
+        {filteredPeople.map(person => (
+          <tr
+            data-cy="person"
+            key={person.slug}
+            className={classNames({
+              'has-background-warning': slug === person.slug,
+            })}
+          >
+            <td>
+              <PersonLink person={person} />
+            </td>
+            <td>{person.sex}</td>
+            <td>{person.born}</td>
+            <td>{person.died}</td>
+            <td>
+              {person.motherName ? (
+                findPersonByName(person.motherName) ? (
+                  <PersonLink
+                    person={findPersonByName(person.motherName) as Person}
+                  />
+                ) : (
+                  person.motherName
+                )
+              ) : (
+                '-'
+              )}
+            </td>
+            <td>
+              {person.fatherName ? (
+                findPersonByName(person.fatherName) ? (
+                  <PersonLink
+                    person={findPersonByName(person.fatherName) as Person}
+                  />
+                ) : (
+                  person.fatherName
+                )
+              ) : (
+                '-'
+              )}
+            </td>
+          </tr>
+        ))}
+
+        {/* {optionsForColumns.map(option => (
+          <td key={option}>
+            {(() => {
+              switch (option) {
+                case ColumnOption.NAME:
+                  return <PersonLink person={person} />;
+
+                case ColumnOption.SEX:
+                  return person.sex;
+
+                case ColumnOption.BORN:
+                  return person.born;
+
+                case ColumnOption.DIED:
+                  return personalbar.died;
+
+                case ColumnOption.MOTHER:
+                  return person
+              }
+            })}
+          </td>
+        ))} */}
```

Comment:
There is a large block of commented-out code. If this code is not needed, consider removing it to keep the codebase clean. If it's for future reference, ensure it's well-documented to avoid confusion.